docs/diginstroom/sip/2.0/1_introduction.md

```diff
@@ -0,0 +1,92 @@
+---
+layout:       default
+title:        Introduction
+parent:       2.0
+grand_parent:  SIP Specification 
+nav_order:    1
+nav_exclude:  false
+---
+Editor's Draft
+{: .label .label-yellow }
+
+# Introduction
+
+This document specifies the meemoo Submission Information Package (SIP), which describes how data and metadata should be packaged when delivered to meemoo for ingest.
+As a digital archive for over 160 content partners, meemoo ingests and disseminates an ever-growing number of digital objects.
+These collections contain a wealth of content and information stored in various types of digital file formats.
+They are accompanied by metadata that is described in a variety of formats.
+Therefore, the current SIP specification was developed to standardize the delivery of (media) content and metadata by meemoo's content partners and increase scalability and sustainability.
+
+The meemoo SIP is a valid [E-ARK SIP](https://earksip.dilcis.eu/) and uses a two-level hierarchical directory structure (_package - representation_) to aggregate and describe media assets, including video, audio, images, captions, etc. At the lowest directory level, the _representation level_, these assets are described in aggregate as digital representations.
+One level higher, the _package_ directory _level_, embodies the represented content or [_intellectual entity_](./3_core-concepts.html), such as the work that is being depicted.
+
+Metadata can occur at every SIP level to add administrative, structural, descriptive, and preservation information about the data and its context.
+Examples are the author of a representation, the author of what the representation represents (i.e. the intellectual entity), or the creation date of a representation.
+Metadata are written down in XML files using the common vocabularies [METS](https://www.loc.gov/standards/mets), [DCMI Metadata Terms](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/), and [PREMIS](https://www.loc.gov/standards/premis/).
+
+The meemoo SIP specification itself cannot be used for actual ingest in the meemoo archive.
+Depending on the type of content, specific mappings are required for ingest.
+These mappings consist of additional requirements on top of the current meemoo SIP specification and are captured in the different content profiles.
+In summary, content that is delivered to meemoo for ingest must always be packaged in a meemoo SIP that adheres to a specific content profile.
+
+## Changes compared to version 1.X
+
+TODO
+
+- EARK compliance
+- no more Bag level
+[BagIt File Packaging Format](https://www.rfc-editor.org/rfc/rfc8493.html)
```

Comment:
Should this be covered here, if it's already (partly) covered in the index.md two levels up?